front/pages/w/[wId]/assistant/gallery.tsx

```diff
@@ -12,7 +12,7 @@ import { useRouter } from "next/router";
 import { useState } from "react";
 
 import { AssistantDetails } from "@app/components/assistant/AssistantDetails";
-import { AssistantPreview } from "@app/components/assistant/AssistantPreview";
```

Comment:
Shouldn't the assiociated be removed since it now comes from sparkle?

Reply:
It's still used in the empty new conversation screen. I'll remove it once I implemented the associated variant.

Reply:
Roger that :+1: